book/run/config.md

```diff
@@ -221,6 +221,19 @@ The storage history indexing stage builds an index of what blocks a particular s
 commit_threshold = 100000
 ```
 
+### `etl`
+
+The storage history indexing stage builds an index of what blocks a particular storage slot changed.
+
+```toml
+[stages.etl]
+# The size of temporary file in bytes for ETL data collector.
```

Comment:
because we do `/ 2` now, it's about the total size of ETL files that the node can allocate, and not just one

Reply:
>  it's about the total size of ETL files 
it's not though, it's the threshold size before creating a new ETL file